br/pkg/pdutil/pd_test.go

```diff
@@ -99,17 +95,19 @@ func (s *testPDControllerSuite) TestGetClusterVersion(c *C) {
 
 	ctx := context.Background()
 	respString, err := pdController.getClusterVersionWith(ctx, mock)
-	c.Assert(err, IsNil)
-	c.Assert(respString, Equals, "test")
+	require.NoError(t, err)
+	require.Equal(t, "test", respString)
 
 	mock = func(context.Context, string, string, *http.Client, string, io.Reader) ([]byte, error) {
 		return nil, errors.New("mock error")
 	}
 	_, err = pdController.getClusterVersionWith(ctx, mock)
-	c.Assert(err, NotNil)
+	require.Error(t, err)
 }
 
-func (s *testPDControllerSuite) TestRegionCount(c *C) {
+func TestRegionCount(t *testing.T) {
+	t.Parallel()
```

Comment:
Test fail here, I will fix it ASAP.

Reply:
I suggest you change `pd_test.go` to `pd_serial_test.go`. Stability over efficiency, go wrong fast make no sense.

Reply:
It is related to wrong predicates... I marked here just to show the test case fails. Sorry for the misleading

Reply:
> I suggest you change pd_test.go to pd_serial_test.go. Stability over efficiency, go wrong fast make no sense.
Shall we split the `TestScheduler` to another file `pd_serial_test.go`?

Reply:
I push a follow up to run all tests in serial and speed up the retry test by failpoints.